core/consensus/grandpa/impl/grandpa_impl.cpp

```diff
@@ -285,6 +285,12 @@ namespace kagome::consensus::grandpa {
     }
   }
 
+  void GrandpaImpl::updateNextRound(RoundNumber round_number) {
+    if (auto round = selectRound(round_number, std::nullopt)) {
+      round->update(true, false, false);
```

Comment:
Just a thought: signature (bool, bool, bool) is VERY obscure - seeing "(true, false, false)" I have no idea what's the intent here. Maybe we could change it to something more expressive? Like enum classes or like?